### PR TITLE
update integration test readme

### PR DESCRIPTION
**Pull request by ailisp**
This is the update I made tests run and pass on my mac, hopefully helpful to others.

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -7,15 +7,18 @@ Running integration tests requires you to have relayer and sandbox docker images
...
 For M1 you may want to pull the following image instead:
 
 ```BASH
+docker pull ghcr.io/near/os-relayer
 docker pull ghcr.io/near/sandbox:latest-aarch64
+docker pull ghcr.io/near/near-lake-indexer:18ef24922fd7b5b8985ea793fdf7a939e57216ba
 ```
 
-In case of authorization issues make sure you have logged into docker using your [access token](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic).
+In case of authorization issues make sure you have logged into docker using your [access token](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic). Your github account should have access to https://github.com/near/pagoda-relayer-rs/pkgs/container/os-relayer, otherwise ask to be added.
```
Comment (volovyks) [inline integration-tests/README.md:21]: @anshal-savla @anthony-near can we make it public?

File: integration-tests/README.md [hunk 2]
```diff
@@ -30,6 +33,11 @@ aws --profile default configure set aws_access_key_id "123"
 aws --profile default configure set aws_secret_access_key "456"
 ```
 
+Build the contract:
+```bash
+cargo build -p mpc-contract --target wasm32-unknown-unknown --release --target-dir target/seperate_wasm
```
Comment (volovyks) [inline integration-tests/README.md:38]: I thought we automated this. cc @ChaoticTempest

**Review by volovyks — APPROVED** [review]
looks good!
Let's clarify a few things before merging.

File: integration-tests/README.md (hunk 2 above)
Comment (ChaoticTempest) [reply integration-tests/README.md:38]: yup, it's no longer necessary to specify `-p mpc-contract` and `--target-dir ...`

Comment (ailisp) [reply integration-tests/README.md]: Yeah, looks it's fixed in latest develop

**Comment by ailisp** [comment]
Looks like this is not relevant after latest refactor.  Fastauth tests are pretty fast and reliable to run, but not the chain-signatures test, which I'll attempt to optimize in a new PR

**Pull request closed without merging**